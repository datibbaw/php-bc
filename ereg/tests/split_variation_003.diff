004+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
004- Error: 8192 - Function split() is deprecated, %s(73)
019+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
019- Error: 8192 - Function split() is deprecated, %s(73)
026+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
027+ array(5) {
026- Error: 8192 - Function split() is deprecated, %s(73)
027- array(1) {
029+   string(1) "1"
030+   [1]=>
031+   string(1) "2"
032+   [2]=>
033+   string(1) "3"
034+   [3]=>
035+   string(1) "4"
036+   [4]=>
037+   string(1) "5"
029-   string(9) "1 2 3 4 5"
033- Error: 8192 - Function split() is deprecated, %s(73)
034- array(1) {
036-   string(9) "1 2 3 4 5"
041+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
042+ array(5) {
041- Error: 8192 - Function split() is deprecated, %s(73)
042- Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
043- NULL
044+   string(1) "1"
045+   [1]=>
046+   string(1) "2"
047+   [2]=>
048+   string(1) "3"
049+   [3]=>
050+   string(1) "4"
051+   [4]=>
052+   string(1) "5"
047- Error: 8192 - Function split() is deprecated, %s(73)
048- Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
049- NULL
050- Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)
052- Arg value Array 
053- Error: 8192 - Function split() is deprecated, %s(73)
054- Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
055- NULL
056- Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)
057- 
058- Arg value Array 
057+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
058+ Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
059+ bool(false)
060+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)
061+ 
062+ Arg value Array 
063+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
064+ Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
065+ bool(false)
066+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)
067+ 
068+ Arg value Array 
069+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
070+ Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
071+ bool(false)
072+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)
073+ 
074+ Arg value Array 
075+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
076+ Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
077+ bool(false)
081+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
082+ Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
083+ bool(false)
084+ 
085+ Arg value  
086+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
087+ array(1) {
088+   [0]=>
089+   string(9) "1 2 3 4 5"
090+ }
091+ 
092+ Arg value  
093+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
094+ array(1) {
095+   [0]=>
096+   string(9) "1 2 3 4 5"
097+ }
098+ 
099+ Arg value 1 
100+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
101+ array(1) {
102+   [0]=>
103+   string(9) "1 2 3 4 5"
104+ }
105+ 
106+ Arg value  
107+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
108+ array(1) {
109+   [0]=>
110+   string(9) "1 2 3 4 5"
111+ }
112+ 
113+ Arg value 1 
114+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
115+ array(1) {
116+   [0]=>
117+   string(9) "1 2 3 4 5"
118+ }
119+ 
120+ Arg value  
121+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
122+ array(1) {
123+   [0]=>
124+   string(9) "1 2 3 4 5"
125+ }
126+ 
127+ Arg value  
128+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
129+ array(1) {
130+   [0]=>
131+   string(9) "1 2 3 4 5"
132+ }
133+ 
134+ Arg value  
135+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
136+ array(1) {
137+   [0]=>
138+   string(9) "1 2 3 4 5"
139+ }
140+ 
141+ Arg value string 
142+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
143+ array(1) {
144+   [0]=>
145+   string(9) "1 2 3 4 5"
146+ }
147+ 
148+ Arg value string 
149+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
150+ array(1) {
151+   [0]=>
152+   string(9) "1 2 3 4 5"
153+ }
154+ Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)
155+ 
156+ Arg value  
157+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
158+ Error: 8 - Object of class stdClass could not be converted to int, /Users/tjerk/Documents/php-bc/ereg/ereg.php(175)
159+ Error: 2 - preg_split() expects parameter 3 to be long, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
160+ bool(false)
161+ 
162+ Arg value  
163+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
164+ array(1) {
165+   [0]=>
166+   string(9) "1 2 3 4 5"
167+ }
169+ Arg value  
170+ Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
171+ array(1) {
172+   [0]=>
173+   string(9) "1 2 3 4 5"
174+ }
175+ Done
059- Error: 8192 - Function split() is deprecated, %s(73)
060- Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
061- NULL
062- Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)
063- 
064- Arg value Array 
065- Error: 8192 - Function split() is deprecated, %s(73)
066- Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
067- NULL
068- 
069- Arg value  
070- Error: 8192 - Function split() is deprecated, %s(73)
071- array(1) {
072-   [0]=>
073-   string(9) "1 2 3 4 5"
074- }
075- 
076- Arg value  
077- Error: 8192 - Function split() is deprecated, %s(73)
078- array(1) {
079-   [0]=>
080-   string(9) "1 2 3 4 5"
081- }
082- 
083- Arg value 1 
084- Error: 8192 - Function split() is deprecated, %s(73)
085- array(1) {
086-   [0]=>
087-   string(9) "1 2 3 4 5"
088- }
089- 
090- Arg value  
091- Error: 8192 - Function split() is deprecated, %s(73)
092- array(1) {
093-   [0]=>
094-   string(9) "1 2 3 4 5"
095- }
096- 
097- Arg value 1 
098- Error: 8192 - Function split() is deprecated, %s(73)
099- array(1) {
100-   [0]=>
101-   string(9) "1 2 3 4 5"
102- }
103- 
104- Arg value  
105- Error: 8192 - Function split() is deprecated, %s(73)
106- array(1) {
107-   [0]=>
108-   string(9) "1 2 3 4 5"
109- }
110- 
111- Arg value  
112- Error: 8192 - Function split() is deprecated, %s(73)
113- Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
114- NULL
115- 
116- Arg value  
117- Error: 8192 - Function split() is deprecated, %s(73)
118- Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
119- NULL
120- 
121- Arg value string 
122- Error: 8192 - Function split() is deprecated, %s(73)
123- Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
124- NULL
125- 
126- Arg value string 
127- Error: 8192 - Function split() is deprecated, %s(73)
128- Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
129- NULL
130- Error: 4096 - Object of class stdClass could not be converted to string, %s(72)
131- 
132- Arg value  
133- Error: 8192 - Function split() is deprecated, %s(73)
134- Error: 2 - split() expects parameter 3 to be long, object given, %s(73)
135- NULL
136- 
137- Arg value  
138- Error: 8192 - Function split() is deprecated, %s(73)
139- array(1) {
140-   [0]=>
141-   string(9) "1 2 3 4 5"
142- }
143- 
144- Arg value  
145- Error: 8192 - Function split() is deprecated, %s(73)
146- array(1) {
147-   [0]=>
148-   string(9) "1 2 3 4 5"
149- }
150- Done