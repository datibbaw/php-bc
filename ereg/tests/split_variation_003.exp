*** Testing split() : usage variations ***

Arg value 10.5 
Error: 8192 - Function split() is deprecated, %s(73)
array(5) {
  [0]=>
  string(1) "1"
  [1]=>
  string(1) "2"
  [2]=>
  string(1) "3"
  [3]=>
  string(1) "4"
  [4]=>
  string(1) "5"
}

Arg value -10.5 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1.07654321E-9 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 0.5 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
NULL

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
NULL

Arg value string 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
NULL

Arg value string 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
NULL
Error: 4096 - Object of class stdClass could not be converted to string, %s(72)

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, object given, %s(73)
NULL

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}
Done