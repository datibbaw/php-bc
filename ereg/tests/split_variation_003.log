
---- EXPECTED OUTPUT
*** Testing split() : usage variations ***

Arg value 10.5 
Error: 8192 - Function split() is deprecated, %s(73)
array(5) {
  [0]=>
  string(1) "1"
  [1]=>
  string(1) "2"
  [2]=>
  string(1) "3"
  [3]=>
  string(1) "4"
  [4]=>
  string(1) "5"
}

Arg value -10.5 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1.07654321E-9 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 0.5 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL
Error: 8 - Array to string conversion, %ssplit_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, array given, %s(73)
NULL

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1 
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
NULL

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
NULL

Arg value string 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
NULL

Arg value string 
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, string given, %s(73)
NULL
Error: 4096 - Object of class stdClass could not be converted to string, %s(72)

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
Error: 2 - split() expects parameter 3 to be long, object given, %s(73)
NULL

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 8192 - Function split() is deprecated, %s(73)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}
Done
---- ACTUAL OUTPUT
*** Testing split() : usage variations ***

Arg value 10.5 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(5) {
  [0]=>
  string(1) "1"
  [1]=>
  string(1) "2"
  [2]=>
  string(1) "3"
  [3]=>
  string(1) "4"
  [4]=>
  string(1) "5"
}

Arg value -10.5 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1.07654321E-9 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(5) {
  [0]=>
  string(1) "1"
  [1]=>
  string(1) "2"
  [2]=>
  string(1) "3"
  [3]=>
  string(1) "4"
  [4]=>
  string(1) "5"
}

Arg value 0.5 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(5) {
  [0]=>
  string(1) "1"
  [1]=>
  string(1) "2"
  [2]=>
  string(1) "3"
  [3]=>
  string(1) "4"
  [4]=>
  string(1) "5"
}
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value string 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value string 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 8 - Object of class stdClass could not be converted to int, /Users/tjerk/Documents/php-bc/ereg/ereg.php(175)
Error: 2 - preg_split() expects parameter 3 to be long, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}
Done
---- FAILED
