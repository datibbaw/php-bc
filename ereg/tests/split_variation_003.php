<?php
/* Prototype  : proto array split(string pattern, string string [, int limit])
 * Description: Split string into array by regular expression 
 * Source code: ext/standard/reg.c
 * Alias to functions: 
 */

function test_error_handler($err_no, $err_msg, $filename, $linenum, $vars) {
	if (error_reporting() != 0) {
		// report non-silenced errors
		echo "Error: $err_no - $err_msg, $filename($linenum)\n";
	}
}
set_error_handler('test_error_handler');
echo "*** Testing split() : usage variations ***\n";

// Initialise function arguments not being substituted (if any)
$pattern = b'[[:space:]]';
$string = '1 2 3 4 5';

//get an unset variable
$unset_var = 10;
unset ($unset_var);

//array of values to iterate over
$values = array(

      // float data
      10.5,
      -10.5,
      10.7654321E-10,
      .5,

      // array data
      array(),
      array(0),
      array(1),
      array(1, 2),
      array('color' => 'red', 'item' => 'pen'),

      // null data
      NULL,
      null,

      // boolean data
      true,
      false,
      TRUE,
      FALSE,

      // empty data
      "",
      '',

      // string data
      "string",
      'string',

      // object data
      new stdclass(),

      // undefined data
      @$undefined_var,

      // unset data
      @$unset_var,
);

// loop through each element of the array for limit

foreach($values as $value) {
      echo "\nArg value $value \n";
      var_dump( split($pattern, $string, $value) );
};

echo "Done";
?>
