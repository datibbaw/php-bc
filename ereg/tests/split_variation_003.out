*** Testing split() : usage variations ***

Arg value 10.5 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(5) {
  [0]=>
  string(1) "1"
  [1]=>
  string(1) "2"
  [2]=>
  string(1) "3"
  [3]=>
  string(1) "4"
  [4]=>
  string(1) "5"
}

Arg value -10.5 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1.07654321E-9 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(5) {
  [0]=>
  string(1) "1"
  [1]=>
  string(1) "2"
  [2]=>
  string(1) "3"
  [3]=>
  string(1) "4"
  [4]=>
  string(1) "5"
}

Arg value 0.5 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(5) {
  [0]=>
  string(1) "1"
  [1]=>
  string(1) "2"
  [2]=>
  string(1) "3"
  [3]=>
  string(1) "4"
  [4]=>
  string(1) "5"
}
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value Array 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 2 - preg_split() expects parameter 3 to be long, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value 1 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value string 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value string 
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/split_variation_003.php(72)

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
Error: 8 - Object of class stdClass could not be converted to int, /Users/tjerk/Documents/php-bc/ereg/ereg.php(175)
Error: 2 - preg_split() expects parameter 3 to be long, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(179)
bool(false)

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}

Arg value  
Error: 16384 - Function split() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(173)
array(1) {
  [0]=>
  string(9) "1 2 3 4 5"
}
Done